packages/core/src/lib/testUtils.ts

```diff
@@ -45,7 +45,7 @@ export const mockWallet = async <Variation extends Wallet>(
         },
         modules: [],
         accounts: [],
-        selectedWalletId: "ledger",
+        selectedWalletId: null,
```

Comment:
We should make `contract` null too

Reply:
I am getting an error (Error: Wallet not signed in) when I set it as null